translations/ru.po

```diff
@@ -84,63 +84,62 @@ msgid "Insert After"
 msgstr "Вставить после"
 
 #: src/config_classes/Formatter.gd
-#, fuzzy
 msgid "Always"
-msgstr "Всегда активно."
+msgstr "Всегда активно"
 
 #: src/config_classes/Formatter.gd
 msgid "All except containers"
-msgstr ""
+msgstr "Все кроме контейнеров"
 
 #: src/config_classes/Formatter.gd
 msgid "Never"
-msgstr ""
+msgstr "Никогда"
 
 #: src/config_classes/Formatter.gd
 msgid "When shorter or equal"
-msgstr ""
+msgstr "Когда короче или равно"
 
 #: src/config_classes/Formatter.gd
 msgid "When shorter"
-msgstr ""
+msgstr "Когда короче"
 
 #: src/config_classes/Formatter.gd
 msgid "3-digit or 6-digit hex"
-msgstr ""
+msgstr "3 или 6 цифровое шестандацитиричное значение"
 
 #: src/config_classes/Formatter.gd
 msgid "6-digit hex"
-msgstr ""
+msgstr "6 цифровое шестандцатяричное значение"
 
 #: src/data_classes/Element.gd
 msgid "{element} must be inside {allowed} to have any effect."
-msgstr ""
+msgstr "{element} должен быть внутри {allowed} чтобы это имело эффект."
 
 #: src/data_classes/ElementG.gd
 msgid "This group has no elements."
-msgstr ""
+msgstr "В этой групе нет элементов."
 
 #: src/data_classes/ElementG.gd
 msgid "This group has only one element."
-msgstr ""
+msgstr "В этой группе только один элемент."
 
 #: src/data_classes/ElementLinearGradient.gd
 msgid "No id attribute defined."
-msgstr ""
+msgstr "Атрибут идентефикатора не определен."
```

Comment:
"id" is the name of the attribute.

Reply:
Oh I see, yeah this string has a version with quotation marks too. That one is valid, and I'll erase this one.